<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>573</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QFrame" name="frameToolButton">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="tBOpen">
         <property name="toolTip">
          <string>Open contact list</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/img/img/document-open.png</normaloff>:/img/img/document-open.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>33</width>
           <height>33</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBNew">
         <property name="toolTip">
          <string>Create new contact list</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/img/img/document-new.png</normaloff>:/img/img/document-new.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>33</width>
           <height>33</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="tBSaveAll">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Save all contact lists</string>
         </property>
         <property name="text">
          <string>Save All</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/img/img/document-save.png</normaloff>:/img/img/document-save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>33</width>
           <height>33</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="tBAbout">
         <property name="toolTip">
          <string>Show about</string>
         </property>
         <property name="text">
          <string>?</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/img/img/face-wink.png</normaloff>:/img/img/face-wink.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>33</width>
           <height>33</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tBHelp">
         <property name="toolTip">
          <string>Instructions</string>
         </property>
         <property name="text">
          <string>???</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/img/img/help-browser.png</normaloff>:/img/img/help-browser.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>33</width>
           <height>33</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
